src/style.css

```diff
@@ -0,0 +1,93 @@
+body {
+  font-family: Avenir, sans-serif;
+}
+
+html {
+  --search-icon-big: url(images/Search.svg) 26px 50% no-repeat;
+  --search-icon-small: url(images/Search.svg) 13px 50% no-repeat;
+  --font-color: #3d4e61;
+  --font-size-small: 14px;
+  --font-size-big: 16px;
+  --linear-gradient-focus: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  --border-input: 1px solid #e1e7ed;
+  --shadow-input: 0 1px 8px 0 #3d4e611a;
+  --shadow-input-hovered: 0 3px 3px 0 #3d4e611a;
+  --background-size-small: 11px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
```

Comment:
look to the checklist, how to corectly connect the fonts

Reply:
+